packages/core/package.json

```diff
@@ -35,7 +35,10 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "@types/uuid": "^8.3.4"
+    "@types/jest": "^27.5.1",
```

Comment:
These should go in the root `package.json` and not in the `core` package as it is a boilerplate dependency.

Reply:
- [ ] move to core